app/Import.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Chompy;
+
+class ImportType {
+  public static $facebook = 'facebook';
```

Comment:
Would you mind adding docblocks for each of these properties and the method before merging? e.g.

```php
/**
 * A Facebook share import type.
 *
 * @var string
 */
public static $facebook = 'facebook';

/**
 * Return the list of valid import types.
 *
 * @return array
 */
public static function all() {
```